tonic/models/vic/vic.py

```diff
@@ -67,17 +67,17 @@ def run(self, global_param, logdir=None, mpi_proc=None):
 
         return self.returncode
 
-    def _call_vic(self, mpi_proc=None, *args):
+    def _call_vic(self, *args, **kwargs):
```

Comment:
I like this approach, however we need to make sure `VIC._call_vic()` gets `**kwargs`, so you'll need to pass those in.